extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSFilter.java

```diff
@@ -90,7 +90,7 @@ public void handle(RoutingContext event) {
                 processRequestedHeaders(response, requestedHeaders);
             }
 
-            boolean allowsOrigin = corsConfig.origins.isEmpty() || corsConfig.origins.contains(origin);
+            boolean allowsOrigin = corsConfig.origins.contains("*") || corsConfig.origins.contains(origin);
```

Comment:
@Postremus Can you please  check it is a single value list too and use `get(0).equals` ? We probably should not allow some random lists which might have non wildcard and wildcard values :-)
There also should be a unit test, please copy and paste one of the tests there to test a `*`